ReactiveCocoaTests/Swift/SignalProducerSpec.swift

```diff
@@ -1044,6 +1044,15 @@ class SignalProducerSpec: QuickSpec {
 
 					expect(completed).to(beTruthy())
 				}
+
+				it("should not deadlock") {
```

Comment:
AFAIK it won't time out (though a CI service might), but you'd see in the console log the test that it's waiting indefinitely on. Pausing in the debugger would reflect a deadlock too.